### PR TITLE
RC form tweaks

### PR DESCRIPTION
**Pull request by mattandrews**
This still needs the arrow button, but free to review otherwise.

This change contains: 

- tweaks to some Unusual Capitalisation In Forms
- adds RC branding/title to pages
- Adds a few fields (project location and additional organisations)

Have at it!

### CODE REVIEW

File: controllers/apply/forms/reaching-communities-form.js [hunk 1]
```diff
@@ -203,40 +225,44 @@ formModel.registerSuccessStep({
...
-            app.render('emails/applicationSummary', {
-                summary: summary,
-                form: formModel,
-                data: flatData
-            }, (errorRenderingTemplate, html) => {
-
-                if (errorRenderingTemplate) {
-                    return reject(errorRenderingTemplate);
-                }
+            app.render(
```
Comment (mattandrews) [inline controllers/apply/forms/reaching-communities-form.js]: Every change to this file below this line is a `prettier`/whitespace tweak, not a proper change.

File: controllers/apply/forms/reaching-communities-form.js [hunk 2]
```diff
@@ -100,70 +100,92 @@ formModel.registerStep({
                             }
                         });
                     }
+                },
+                {
+                    type: 'text',
+                    name: 'project-location',
+                    label: "In your own words, describe the location(s) that you'll be running your project(s) in",
+                    placeholder: 'eg. "Newtown community centre" or "Alfreton, Derby and Ripley"',
```
Comment (davidrapson) [inline controllers/apply/forms/reaching-communities-form.js]: It's a little odd to have a Newtown as an example when this programme is for England only.

File: controllers/apply/forms/reaching-communities-form.js [hunk 3]
```diff
@@ -100,70 +100,92 @@ formModel.registerStep({
...
                     validator: function(field) {
                         return check(field.name)
                             .trim()
                             .not()
                             .isEmpty()
                             .withMessage('Organisation must be provided');
                     }
+                },
+                {
+                    type: 'text',
```
Comment (davidrapson) [inline controllers/apply/forms/reaching-communities-form.js]: Does this do anything yet?

Comment (mattandrews) [reply controllers/apply/forms/reaching-communities-form.js]: How do you mean? It outputs an `<input type="text" />` field...?

File: assets/sass/components/buttons.scss [hunk 4]
```diff
@@ -249,6 +249,22 @@ $btnConf: (
     }
 }
 
+
+
+.btn--next {
+    background-image: arrow-btn();
```
Comment (davidrapson) [inline assets/sass/components/buttons.scss]: Thinking about it, maybe css clip-paths are the way to go here. e.g. http://bennettfeely.com/clippy/
That's the way all the wonky stuff on https://membership.theguardian.com/ was done.

**Comment by davidrapson** [comment]
👍 Changes looks good, parameter store stuff makes sense. Would be good to come back and get some more tests in for this code, but good to merge.

File: assets/sass/components/buttons.scss (hunk 4 above)
Comment (mattandrews) [reply assets/sass/components/buttons.scss]: Just looking into this. Here's the best I could manage:
image

(using `clip-path: polygon(0% 0%, 80% 0%, 95% 50%, 80% 100%, 0% 100%)`)

The original one, though, has a rounded edge on the arrow:

image

[Apparently](https://stackoverflow.com/questions/31765345/how-to-round-out-corners-when-using-css-clip-path), though, you can't easily round corners using CSS clip-path (other than literally draw each point in the corner). Not sure how much of a compromise we can make here.

**Pull request merged**